components/vf-search-client-side/vf-search-client-side.njk

```diff
@@ -7,14 +7,13 @@
   {% set search_client_button_text = context.search_client_button_text %}
   {% set search_client_script = context.search_client_script %}
 
-
   {% set id = context.id %}
   {% set modifiers = context.modifiers %}
   {% set override_class = context.override_class %}
 {% endif -%}
 
-<div class="vf-grid">
-  <form action="{{ search_client_action }}" method="GET" class="vf-form | vf-search vf-search--inline">
+<form action="{{ search_client_action }}" method="GET" class="vf-form | vf-search vf-search--inline">
+  <div class="vf-container__inner">
```

Comment:
I spotted this in the VF today - and it felt like I implemented something 'half baked'. So this might change :D